src/javascripts/_er-diagram.js

```diff
@@ -238,4 +241,8 @@ mumuki.load(function () {
     $('.mu-erd').renderERD();
   });
 
+  //Redraw on bootstrap tab change event
+  $('a[data-toggle="tab"]').on('shown.bs.tab', function () {
```

Comment:
What if the tabs are not related with the ERD? Will this flick or produce any inappropriate behaviour?

Reply:
I tested it out on tabs without any ERDs and it didn't seem to behave any differently, the function is a no-op if no proper SVG or ERD elements are found